apps/src/code-studio/components/SoundPicker.jsx

```diff
@@ -80,41 +81,46 @@ export default class SoundPicker extends React.Component {
     let title = <p>{i18n.chooseSounds()}</p>;
 
     modeSwitch = (
-      <div>
+      <div id="modeSwitch">
         <p onClick={this.setSoundMode} style={headerStyles.soundModeToggle}>
           {i18n.soundLibrary()}
         </p>
         <p onClick={this.setFileMode} style={headerStyles.fileModeToggle}>
           {i18n.makeNewSounds()}
         </p>
-        <hr style={styles.divider} />
       </div>
     );
 
-    const displayFilesTab = this.state.mode === MODE.files;
-    const body = displayFilesTab ? (
-      <AssetManager
-        assetChosen={this.props.assetChosen}
-        assetsChanged={this.props.assetsChanged}
-        allowedExtensions={audioExtension}
-        uploadsEnabled={this.props.uploadsEnabled}
-        useFilesApi={this.props.useFilesApi}
-        projectId={this.props.projectId}
-        soundPlayer={this.props.soundPlayer}
-      />
-    ) : (
-      <SoundLibrary assetChosen={this.getAssetNameWithPrefix} />
-    );
-
+    const displaySoundsTab = this.state.mode === MODE.sounds;
```

Comment:
Naming nit: `displaySoundsTab` doesn't clarify if we're displaying the user uploaded sound files or the sound library.